packages/utilities/src/shadowDom/contexts/MergeStylesDefaultContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { DEFAULT_SHADOW_CONFIG, ExtendedCSSStyleSheet } from '@fluentui/merge-styles';
+import * as React from 'react';
+
+import type { AdoptedStylesheetExHook, AdoptedStylesheetHook } from '../hooks/useAdoptedStylesheet';
+import type { ShadowConfigHook } from '../hooks/useShadowConfig';
+import type {
+  HasMergeStylesShadowRootContextHook,
+  MergeStylesShadowRootContetHook,
```

Comment:
is this a typo?

```suggestion
  MergeStylesShadowRootContentHook,
```

Reply:
It is but, sadly, it was already part of the public API so we're stuck with it.

Reply:
Actually, in fixing up the missing exports I realized this type was never properly exported (it was referenced but the type itself was not exported so it was not usable) so I think it's safe to rename.